resources/subjectscheme.ditamap

```diff
@@ -25,4 +25,59 @@
     <attributedef name="platform"/>
     <subjectdef keyref="os"/>
   </enumerationdef>
+  <!-- Start: Controlling @outputclass values for various elements -->
+  <enumerationdef>
+    <elementdef name="codeblock"/>
+    <attributedef name="outputclass"/>
+    <subjectdef keys="bootstrap">
```

Comment:
None of these are actually Bootstrap classes, so we should probably use `codeblock-outputclass` here.

- The first 3 are DITA-OT–specific [extensions for codeblock processing](https://www.dita-ot.org/dev/reference/extended-functionality.html#code-reference__normalize-codeblock-whitespace)
- The latter 6 are keywords used to trigger syntax highlighting via [Rouge](http://rouge.jneen.net) or [Prism](https://prismjs.com), which would probably be better defined in a separate `<subjectdef>` element or removed from the source files pending selection of a syntax highlighting approach.

Reply:
I changed `<subjectdef keys="codeblock-outputclass"/>` and grouped the values as suggested, but that has no impact on what's available in an authoring UI because the values are still available on `<codeblock>`. It's not clear to me from this comment whether that is the intended goal by separating them. To separate the lists `<codeblock>` would need to be specialized into a different name.

Reply:
I would like to see the `<subjectdef keys="codeblock-languages">` values to be made consistent (i.e., `language-*`), which would require updates to the source files. That may also provide an opportunity to add `language-html` and/or remove some of the markup from the source files as you suggested. Thoughts?

Reply:
Separating the values for `<codeblock>` into separate `<subjectdef>` elements would allow us to distinguish the DITA-OT–specific values from those that trigger syntax highlighting. Though the result set would be identical from an authoring perspective, it would make the map _(and thus the intention)_ more readable.

Ideally, the latter group would be read via `<subjectref>` from a separate map that includes all of the permissible values for the syntax highlighting system we settle on, so that can be easily updated (or regenerated) if/when the upstream project extends the list of permissible values.

We should treat that as a separate issue, and remove all obsolete highlighting classes in the process.